.github/workflows/v2.test.yml

```diff
@@ -0,0 +1,19 @@
+name: VCDM v2 Test Suite
+
+on: [push]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Begin CI...
+        uses: actions/checkout@v2
+      - name: Use Node 16
+        uses: actions/setup-node@v1
```

Comment:
use node v3? and checkout v3? also might be a good idea to use cache with it to speed it up.

Reply:
Do you have a snippet you would prefer I use here?

Reply:
See any of the workflows in trace-interop:
```
...
    steps:
      # Check out repo, set up node, and install dependencies.
      # @see https://github.com/actions/setup-node#usage
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v3
        with:
          node-version: 16.15.1
          cache: 'npm'
      - run: npm ci
...
```